### ISSUE TITLE
Utah Roads: Shapefile link missing/broken

### ISSUE THREAD

**Issue by matthewbarnett86** [agrc/gis.utah.gov#1325]
## Issue description

Hello! I am trying to access the Utah roads shapefile for a school assignment, but the link has disappeared from the page and the link to the shapefile is broken on the https://opendata.gis.utah.gov/datasets/utah-roads page. Even if I could access an old version I would be extremely grateful! *prays*

Any assistance accessing this data would be greatly appreciated!

Best wishes,
Matt

#### What page is the problem visible

https://gis.utah.gov/data/transportation/roads-system/

**Comment by steveoh** [comment]
Paging @gregbunce

**Comment by nathankota** [comment]
Greg is down for the count with influenza b. Can someone else help with this @ZachBeck @MichaelAGRC ?

**Comment by ZachBeck** [comment]
It seems like it's working...

**Comment by nathankota** [comment]
@matthewbarnett86 - I can see the shapefile download is struggling, but the file geodatabase download seems to be working. Will that work for you while we look into the shapefile issue?

**Comment by steveoh** [comment]
@matthewbarnett86, the reason the roads shapefile link was removed was because of a bug in the esri open data download system. It was making the 860MB roads larger than 2GB during their export process which is not valid for shapefiles. While that bug is being worked out by the upstream folks, we removed the option since the file geodatabase, geojson, and the geoservice are still available. 

We will need to provide the shapefile via an alternate method for the time being since esri is telling me it will take 2-4 weeks to clean up their mess.
